python/custreamz/custreamz/kafka.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+import confluent_kafka as ck
+from cudf_kafka._lib.kafka import KafkaDatasource
+
+import cudf
+
+from custreamz.utils import docutils
+
+
+# Base class for anything class that needs to interact with Apache Kafka
+class CudfKafkaClient(object):
+    def __init__(self, kafka_configs):
+        self.kafka_configs = kafka_configs
+
+        self.kafka_confs = {}
+        for key, value in self.kafka_configs.items():
+            self.kafka_confs[str.encode(key)] = str.encode(value)
```

Comment:
Can use a dictionary comprehension here instead:
```suggestion
        self.kafka_confs = {str.encode(key): str.encode(value) for key, value in self.kafka_configs.items()}
```